### PR TITLE
What's New in Redpanda Cloud

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/2081

This will be a new page in our doc to coincide with the Nov Cloud release, similar to our existing [What's New for Redpanda platform](https://docs.redpanda.com/current/get-started/whats-new/). As the page grows with releases, it'll look similar to [this](https://cloud.google.com/architecture/framework/whats-new)), with the release date appears in the right pane. 

preview: https://deploy-preview-117--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/

### CODE REVIEW

**Comment by micheleRP** [comment]
@bpraseed, @towfiqa, @vladoschreiner: 
- Does Redpanda Cloud have a release notes we can link to, like https://github.com/redpanda-data/redpanda/releases/tag/v23.2.1[this^]?
- [Customer-managed VPC](https://docs.redpanda.com/current/deploy/deployment-option/cloud/vpc-byo-gcp/) is not listed on the feature list. Should we include it here? 
- I'll add a link to the Using Schema Registry in the UI soon, after it merges.

**Comment by Feediver1** [comment]
I could see the right sidebar growing quite lengthy in a short period of time. Perhaps just have the Cloud release date there/adjust heading levels so that the individual features are not listed?

**Comment by bpraseed** [comment]
> [Customer-managed VPC](https://docs.redpanda.com/current/deploy/deployment-option/cloud/vpc-byo-gcp/) is not listed on the feature list. Should we include it here?
Let not include as we announce this already and is in docs.

We should add AWS arm support for BYOC.

for Maintainance window : Customer cant select any window atm but select a specific time during the window. I would edit to "Redpanda Cloud provides the flexibility to select a Maintenance window to apply updates during off-hours at a time that best suits your business. Through your Redpanda Cloud cluster setting, customers are able to specify a Maintenance window during which Redpanda services can be upgraded or patched to a new version" cc: @towfiqa for any additional word smithing.

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+= What's New in Redpanda Cloud
+:description: Summary of new features in Redpanada Cloud releases.
+
+This topic lists new features added in Redpanda Cloud. 
+
+== November 2023
+
+==== Optimized BYOC tiers support ARM-based AWS Graviton processors
+
+BYOC clusters now support ARM-based AWS Graviton processors on Amazon EC2. This allows decreased VM costs and increased partition support.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc]:
```suggestion
BYOC clusters now support ARM-based AWS Graviton processors on Amazon EC2. This decreases VM costs and increases partition support.
```

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+==== Iceberg Sink connector
+
+With the xref:deploy:deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc[managed connector for Apache Iceberg], you can write data into Iceberg tables. This enables integration with the data lake ecosystem and efficient data management for complex analytics.
+
+==== Schema Registry management
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc:16]: Are there updated docs for SR? If so, should link to them.

**Review by Feediver1 — APPROVED** [review]

**Comment by bpraseed** [comment]
> BYOC clusters now support ARM-based AWS Graviton processors on Amazon EC2. This allows decreased VM costs and increased partition support.

Can we change this to
" AWS BYOC clusters now support ARM-based Graviton2 instances. This allows lower VM costs and supports increased partition count "

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc:16]: thanks, I added a link to the new doc!

**Review by bpraseed — APPROVED** [review]

**Pull request merged**